slash/css/src/Card/Card.css

```diff
@@ -82,3 +82,21 @@
   --card-text-color: var(--red30);
   --card-icon-color: var(--red30);
 }
+
+.af-card--active {
```

Comment:
on ne pouvait pas juste mettre un not(:disabled) ?
```suggestion
 .af-card--active:not(:disabled) {
```

Reply:
si je le mets comme cela, on n'a plus de carte active et disabled en même temps (en terme de design).

Ce qu'on voudrait : 

<img width="256" height="198" alt="Image" src="https://github.com/user-attachments/assets/85ea063d-ce56-4c39-bc39-682d6ce78d8c" />

Ce qu'on aurait : 

<img width="253" height="224" alt="Image" src="https://github.com/user-attachments/assets/7231f493-e189-4c9e-a879-01b347b38623" />